tests/test_sklearn_decision_tree_converters.py

```diff
@@ -33,56 +33,57 @@ def test_random_forest_implementation(self):
 
-    # Perf_tree_trav classifier
+    # Random forest perf_tree_trav classifier
     def test_random_forest_perf_tree_trav_classifier_converter(self):
-        self._run_random_forest_classifier_converter(2, extra_config={"tree_implementation": "perf_tree_trav"})
+        self._run_tree_classification_converter(RandomForestClassifier, 2, extra_config={"tree_implementation": "perf_tree_trav"}, n_estimators=10)
 
-    # Multi classifier
+    # Random forest multi classifier
     def test_random_forest_multi_classifier_converter(self):
-        self._run_random_forest_classifier_converter(3)
+        self._run_tree_classification_converter(RandomForestClassifier, 3, n_estimators=10)
 
-    # Gemm multi classifier
+    # Random forest gemm multi classifier
     def test_random_forest_gemm_multi_classifier_converter(self):
-        self._run_random_forest_classifier_converter(3, extra_config={"tree_implementation": "gemm"})
+        self._run_tree_classification_converter(RandomForestClassifier, 3, extra_config={"tree_implementation": "gemm"}, n_estimators=10)
 
-    # Tree_trav multi classifier
+    # Random forest tree_trav multi classifier
     def test_random_forest_tree_trav_multi_classifier_converter(self):
-        self._run_random_forest_classifier_converter(3, extra_config={"tree_implementation": "tree_trav"})
+        self._run_tree_classification_converter(RandomForestClassifier, 3, extra_config={"tree_implementation": "tree_trav"}, n_estimators=10)
 
-    # Perf_tree_trav multi classifier
+    # Random forest perf_tree_trav multi classifier
     def test_random_forest_perf_tree_trav_multi_classifier_converter(self):
-        self._run_random_forest_classifier_converter(3, extra_config={"tree_implementation": "perf_tree_trav"})
+        self._run_tree_classification_converter(RandomForestClassifier, 3, extra_config={"tree_implementation": "perf_tree_trav"}, n_estimators=10)
 
-    # Shifted classes
+    # Random forest shifted classes
     def test_random_forest_classifier_shifted_labels_converter(self):
-        self._run_random_forest_classifier_converter(3, labels_shift=2, extra_config={"tree_implementation": "gemm"})
-        self._run_random_forest_classifier_converter(3, labels_shift=2, extra_config={"tree_implementation": "tree_trav"})
-        self._run_random_forest_classifier_converter(3, labels_shift=2, extra_config={"tree_implementation": "perf_tree_trav"})
+        self._run_tree_classification_converter(RandomForestClassifier, 3, labels_shift=2, extra_config={"tree_implementation": "gemm"}, n_estimators=10)
+        self._run_tree_classification_converter(RandomForestClassifier, 3, labels_shift=2, extra_config={"tree_implementation": "tree_trav"}, n_estimators=10)
+        self._run_tree_classification_converter(RandomForestClassifier, 3, labels_shift=2, extra_config={"tree_implementation": "perf_tree_trav"}, n_estimators=10)
```

Comment:
Should we separate these into 3 different tests?

Reply:
Yes please!

Reply:
Doing it now. Do you feel anything else is off that we should address?